dags/public-health/Code75.py

```diff
@@ -56,7 +57,7 @@ def update_code_75(**kwargs):
     engine.connect().execute(create_table_statement)
 
     # Deleting old records
-    object_id_list = ",".join(list(sdf["OBJECTID"].astype(str)))
+    object_id_list = ",".join(list(sdf["OBJECTID"]))
```

Comment:
Turns out I was totally off-base, sorry @brendanbailey! This type cast is necessary for the string concatenation to work. Then I think this is good to go!